pandora-client-web/src/components/wardrobe/wardrobeItems.tsx

```diff
@@ -101,10 +104,14 @@ export function WardrobeItemManipulation({ className }: { className?: string; })
 	const currentFocus = useObservable(focus);
 	const { preFilter, containerContentsFilter, assetFilterAttributes } = useWardrobeItems(currentFocus);
 
-	const title: string = target.type === 'character' ? 'Currently worn items' : 'Room inventory';
+	const appearance = useWardrobeTargetItems(target);
+	const itemCount = useMemo(() => AppearanceItemsCalculateTotalCount(appearance), [appearance]);
+	const itemLimit: number = target.type === 'character' ? ITEM_LIMIT_CHARACTER_WORN : ITEM_LIMIT_ROOM_INVENTORY;
+	const titlePrefix: string = target.type === 'character' ? 'Currently worn items' : 'Room inventory';
+
+	const title = `${ titlePrefix }, used: ${ itemCount } / ${ itemLimit } (${100 * itemCount / itemLimit}%)`;
```

Comment:
Same thing about the ceil:
```suggestion
	const title = `${ titlePrefix }, used: ${ itemCount } / ${ itemLimit } (${  Math.ceil(100 * itemCount / itemLimit) }%)`;
```